DataConnectors/AWS-S3/ConfigCustomLogDataConnector.ps1

```diff
@@ -0,0 +1,44 @@
+# ***********       Main Flow       ***********
+
+# Validate AWS configuration
+Test-AwsConfiguration
+
+Write-Log -Message "Starting C data connector configuration script" -LogFileName $LogFileName -Severity Verbose
+Write-Log -Message "This script creates an Assume Role with minimal permissions to grant Azure Sentinel access to your logs in a designated S3 bucket & SQS of your choice, enable S3 bucket, SQS Queue, and S3 notifications." -LogFileName $LogFileName -LinePadding 2
+Write-ScriptNotes
+
+New-ArnRole
+Write-Log -Message "Executing: aws iam get-role --role-name $roleName" -LogFileName $LogFileName -Severity Verbose
+$roleArnObject = aws iam get-role --role-name $roleName
+$roleArn = ($roleArnObject | ConvertFrom-Json ).Role.Arn
+Write-Log -Message $roleArn -LogFileName $LogFileName -Severity Verbose
+
+# Create S3 bucket for storing logs
+New-S3Bucket
+
+Write-Log -Message "Executing: (aws sts get-caller-identity | ConvertFrom-Json).Account" -LogFileName $LogFileName -Severity Verbose
+$callerAccount = (aws sts get-caller-identity | ConvertFrom-Json).Account
+Write-Log -Message $callerAccount -LogFileName $LogFileName -Severity Verbose
+
+
```

Comment:
Is it possible that the customer want KMS for the custom logs?

Reply:
Alaa told me not to add KMS for now